src/sentry/models/groupsubscription.py

```diff
@@ -178,7 +178,8 @@ class GroupSubscription(Model):
 
     project = FlexibleForeignKey("sentry.Project", related_name="subscription_set")
     group = FlexibleForeignKey("sentry.Group", related_name="subscription_set")
-    user_id = HybridCloudForeignKey(settings.AUTH_USER_MODEL, on_delete="CASCADE")
+    user_id = HybridCloudForeignKey(settings.AUTH_USER_MODEL, null=True, on_delete="CASCADE")
+    team_id = HybridCloudForeignKey("sentry.Team", null=True, db_index=True, on_delete="CASCADE")
```

Comment:
@isabellaenriquez since this is in the region silo, I think we can just use `FlexibleForeignKey` instead of `HybridCloudForeignKey`

Reply:
Yes, Steve is correct, `HybridCloudForeignKey` should only be used when a relationship spans regions. You want `FlexibleForeignKey` here.